vagrant/README.md

```diff
@@ -0,0 +1,79 @@
+# About
+
+This is a fairly minimal Vagrantfile and associated bootstrap scripts for
+setting up a GCC-based build environment for
+[TrustedFirmware-M](https://www.trustedfirmware.org).
+
+Note that the vagrant virtual machine will contain an independent copy of TF-M
+and the mbed-os-tf-m-regression-tests.
+
+Note that by default vagrant shares `/vagrant` in the virtual machine with the
+host. You can use this directory to copy binaries from inside the vagrant
+machine out to the host for e.g. programming via USB.
+
+# Howto
+
+### TF-M + Mbed OS Regression Tests
+
+The following quickstart instructions will build you the TF-M regression tests
+for MUSCA_B1 on Mbed OS.
+
+```
+$ cd vagrant
+$ vagrant up
+$ vagrant ssh
+$ cd mbed-os-tf-m-regression-tests
+$ mbed deploy
+$ python3 build_tfm.py -m CY8CKIT_064S2_4343W -t GNUARM -c ConfigRegressionIPC.cmake
```

Comment:
Please remove this target - rename to ARM_MUSCA_B1

Reply:
Done